src/main/java/list/components/CardItem.java

```diff
@@ -0,0 +1,60 @@
+package list.components;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+import java.util.List;
+
+public class CardItem extends BaseComponents {
+    private WebElement superElm;
+    @FindBy(xpath = ".//div[@class='at']")
+    private WebElement location;
+    @FindBy(xpath = ".//div[@class='p']")
+    private WebElement price;
+    @FindBy(xpath = ".//div[@class='clabel']")
+    private WebElement label;
+    @FindBy(xpath = ".//div[@class='l']")
+    private WebElement description;
+
+
+    public CardItem(WebElement element) {
+        super(element);
+
+    }
+
+
+    public WebElement getPrice() {
```

Comment:
Please don't return WebElement, as per POM you shouldn't have access to WebElements from tests.